presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergLogicalPlanner.java

```diff
@@ -112,6 +113,54 @@ protected QueryRunner createQueryRunner()
         return createIcebergQueryRunner(ImmutableMap.of("experimental.pushdown-subfields-enabled", "true"), ImmutableMap.of());
     }
 
+    @Test
+    public void testFiltersWithPushdownDisable()
+    {
+        Session sessionWithoutFilterPushdown = pushdownFilterDisabled();
+        assertUpdate("CREATE TABLE test_filters_with_pushdown_disable(id int, name varchar, r row(a int, b varchar)) with (partitioning = ARRAY['id'])");
+
+        // Only identity partition column predicates, would be enforced totally by tableScan
+        assertPlan(sessionWithoutFilterPushdown, "SELECT name, r FROM test_filters_with_pushdown_disable WHERE id = 10",
```

Comment:
We can use `com.facebook.presto.tests.AbstractTestQueryFramework#assertPlan(java.lang.String, com.facebook.presto.sql.planner.assertions.PlanMatchPattern)` which uses default session.